tools/codegen/gofilegen/schema/schema_file_test.go

```diff
@@ -162,73 +121,7 @@ func TestSchemaGenerationFromCodeSpec(t *testing.T) {
 			withObjType:    true,
 			goldenFileName: "primitive-attributes",
 		},
-		"Nested attributes": {
```

Comment:
nit: Given custom type is how we now manage all nested attributes, would consider renaming the existing test `Custom type attributes` to `Nested attributes` and just ensuring we are capturing the existing nested attributes scenarios.